apps/mark-scan/frontend/src/app_root.tsx

```diff
@@ -439,6 +440,18 @@ export function AppRoot({
     return <JamClearedPage stateMachineState={stateMachineState} />;
   }
 
+  const isInPaperLoadingStage =
+    stateMachineState === 'accepting_paper' ||
+    stateMachineState === 'loading_paper';
```

Comment:
In the second video, it seemed a little weird to me that the "start voting" screen is shown while the paper is still being loaded and then if the poll worker removes their card, this error can occur which makes them have to start over. I feel like the "start voting" screen indicates that it's safe to remove the poll worker card. Maybe a "loading paper... please wait" screen would be helpful?

Going to hold off on technical review for now til we resolve the product discussion.

Reply:
Cool, just let me know when you want me to take another look

Reply:
Uploaded the new video and re-requested review 🙏

Reply:
One more UX question: Is there a reason to have a delay and show a message when the poll worker card is removed before loading paper? Kind of seems like it could just cancel and go back to the Insert Card screen immediately

Reply:
No strong reason - I kept them the same for consistency but we can also just end the session. The more I think about it, it's probably less confusing if we skip the message.

To be clear, are you ok with keeping the message for when the poll worker card is pulled during the physical paper load stage?

Reply:
I'm ok with keeping it, though I'd be curious to see how it feels to you without it.

Reply:
I think in that case, it'll be more surprising to the poll worker if we end the session. IMO we should leave the message in for the actual paper loading stage, or remove both the message and auth check (so a poll worker can pull their card once the paper has been grasped by the printer).

Of those 2 options, the former is safer, so inclined to leave it as is.

Reply:
Sounds good to me!